scraper_common.js

```diff
@@ -75,7 +75,16 @@ async function execute(usePuppeteer, scrapers) {
                 startTime,
                 numberAppointments
             );
+
+            // Add the current timestamp to each individual location
+            const nowTimestamp = moment().format();
```

Comment:
We've had it both ways, but now we just have one. But it seemed silly to me to use `moment().format()` when we could just use `new Date()`.

Reply:
@livgust changed them all to `moment().format()`

Reply:
@harcod made the great point that we should be consistent, and moment() has a better library than Date() so here we are :)